src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -183,6 +183,7 @@ def load_arguments(self, _):
         c.argument('slot', options_list=['--slot', '-s'], help='slot to restore the deleted content to')
         c.argument('restore_content_only', action='store_true',
                    help='restore only deleted files without web app settings')
+        c.argument('target_app_svc_plan', options_list=['--target-app-svc-plan'], help='The app service plan for the new azure web app.')
```

Comment:
`--target-app-svc-plan` is the default option, please do not explicitly define it repeatedly
```suggestion
        c.argument('target_app_svc_plan', help='The app service plan for the new azure web app.')
```

Reply:
Thanks for the suggestion @zhoxing-ms , Fixed it.